content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -202,6 +202,16 @@ An [SQS Queue][32] (`aws_sqs_queue`) is considered publicly accessible if:
 
 See [Amazon SQS security best practices][33] for more information about public SQS queues.
 
+### Amazon Lambda Function
+
+A [lambda function][58] (`aws_lambda_function`) is considered publicly accessible if:
```

Comment:
```suggestion
A [Lambda function][58] (`aws_lambda_function`) is considered publicly accessible if:
```

Reply:
Following the nomenclature, I should capitalize both letters.

Reply:
Thanks for following up. Could you clarify what nomenclature you're referring to?

For additional context about why I've made this request:
 - The [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#headers) requests that level-3 headers use sentence case
 - `Lambda function` is used in AWS documentation (see [examples on this page](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html))
 - `Lambda function` is what's used in other pages of the documentation site (see [examples on this page](https://docs.datadoghq.com/serverless/aws_lambda/installation/python/?tab=datadogcli))

Reply:
Thanks for following up, I think I accidentally deleted my previous comment. Could you clarify what nomenclature you're referring to?

For additional context about why I'm making this request:
  - The [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#headers) requests sentence case for level-3 headers
  - AWS documentation mostly uses `Lambda function` (see [this page](https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html) for examples)
  - Our documentation mostly uses `Lambda function` (see [this page](https://docs.datadoghq.com/serverless/aws_lambda/logs/?tab=serverlessframework) for examples)

Reply:
Thanks ! Following your PR, I made the changes that you suggested